dora/core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerProcess.java

```diff
@@ -131,13 +130,14 @@ public final class AlluxioWorkerProcess implements WorkerProcess {
       mRpcConnectAddress = requireNonNull(dataServerFactory.getConnectAddress());
       mStartTimeMs = System.currentTimeMillis();
       List<Callable<Void>> callables = ImmutableList.of(() -> {
-        if (worker instanceof BlockWorker) {
-          mRegistry.add(BlockWorker.class, worker);
-        } else if (worker instanceof DoraWorker) {
+        if (worker instanceof DoraWorker) {
           mRegistry.add(DoraWorker.class, worker);
+          mRegistry.addAlias(DataWorker.class, worker);
```

Comment:
this `DataWorker` is a bit redundant to me, let's merge `DataWorker` into `DoraWorker`?